app/helpers/comments_helper.rb

```diff
@@ -64,6 +64,17 @@ def like_button_text(comment)
     end
   end
 
+  def contextual_comment_url(comment, article: nil)
```

Comment:
Contextual because the resulting URL will depend on the context (signed in/out user). If anyone feels there's a better name for this method I'd be happy to rename it, otherwise okay with keeping it this way 😄

Reply:
I love this name, very communicative.